sphinx/templates/quickstart/conf.py_t

```diff
@@ -108,6 +108,10 @@ html_static_path = ['{{ dot }}static']
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
 }
+
+# Prevent accidental intersphinx resolution for labels, documents, and other
+# basic cross-references.
+intersphinx_disabled_domains = ['std']
```

Comment:
I think this should be set as a default setting of `intersphinx_disabled_domains` instead of the generated conf.py. I guess you tried not to introduce a breaking change. But this implementation is not good for me.

Reply:
I agree, and indeed I wanted to avoid breaking changes to get the functionality released earlier.  My idea was to make a another PR after this, but against master, that changes the default value and removes this bit again. Would that be fine?
Alternatively I can also just remove this now and still make the default value change in master?

Reply:
Quickstart stuff removed.